test/generic/RationalFunctionField-test.jl

```diff
@@ -6,6 +6,7 @@ end
 @testset "Generic.RationalFunctionField.constructors" begin
    # Univariate
 
+   T, x = rational_function_field(QQ)
```

Comment:
This looks like a copy&paste accident at first glance, if it was intentional, maybe add a comment?
```suggestion
   T, x = rational_function_field(QQ)  # verify default variable name is provided
```
Then again in line 79 this is already tested so it seems a bit redundant here?

Reply:
I like to make it explicit